tests/ciscotest.rb

```diff
@@ -113,4 +113,26 @@ def interfaces_id
     end
     @@interfaces_id
   end
+
+  # Remove all router bgps.
```

Comment:
I'd prefer to have an intermediate 'bgptest.rb' class (TestBgpAF < BgpTest < CiscoTest < TestCase) rather than putting feature-specific code in ciscotest.rb - but not critical.

Alternately, would it make sense to add these methods to the NU classes bgp.rb and vrf.rb?